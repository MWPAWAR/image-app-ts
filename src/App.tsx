import { FC } from 'react'
import { Routes, Route, Navigate } from 'react-router-dom'
import ImageListContainer from './components/ImageListContainer/ImageListContainer'

const App: FC = () => (
  <Routes>
    <Route
      path="/recently-added"
      element={<ImageListContainer activeTab="recentlyAdded" />}
    />
    <Route
      path="/favorites"
      element={<ImageListContainer activeTab="favorites" />}
    />
    <Route path="/" element={<Navigate to="/recently-added" />} />
  </Routes>
)

export default App
