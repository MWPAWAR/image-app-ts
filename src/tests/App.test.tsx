import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import App from '../App'

describe('App', () => {
  it('matches snapshot correctly', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    )
    expect(tree).toMatchSnapshot()
  })
})
