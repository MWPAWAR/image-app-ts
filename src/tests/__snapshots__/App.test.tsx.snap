// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot correctly 1`] = `null`;
